scripts/qa_crowbarsetup.sh

```diff
@@ -4029,7 +4029,7 @@ EOH
             fi
         fi
 
-        nosetests testsuites/testcloud_sanity.py
+        nosetests testsuites/depricated/testcloud_sanity.py
```

Comment:
I have seen the error in an mkcloud run, but I don't know why this should fix it. The file seems to be there: https://gitlab.suse.de/ceph/qa-automation/blob/master/testsuites/testcloud_sanity.py .

Also that `depricated` word looks like a typo.